src/screens/Auth/LoginStack/PasswordResetScreen/PasswordResetScreen.tsx

```diff
@@ -2,39 +2,71 @@ import React, { useState } from 'react'
 import {
   Text,
   View,
-  Alert,
-  TouchableOpacity,
 } from 'react-native'
 import styles from './styles'
 import BlackHeading from '../../../../shared_components/BlackHeading/BlackHeading'
 import GreyTextInput from '../../../../shared_components/GreyTextInput/GreyTextInput'
 import PinkButton from '../../../../shared_components/PinkButton/PinkButton'
+import { NavigationScreenProp, NavigationState } from 'react-navigation'
+import { BASE_PATH } from 'react-native-dotenv'
 
-export default function PasswordResetScreen() {
+interface Props {
+  navigation: NavigationScreenProp<NavigationState>,
+}
+
+export default function PasswordResetScreen(props: Props) {
   const [email, setEmail] = useState('')
+  const [message, setMessage] = useState('')
+  const [messageStyle, setMessageStyle] = useState(styles.messageError)
+  const {navigate} = props.navigation 
+
+  async function checkEmail(){
+    fetch(`${BASE_PATH}/api/auth/forgot-password`, {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        'Email': email,
+        'Full name': '',
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        console.log(data)
+        if(data.statusCode == 404) {
+          setMessage('Error: Email is not registered to a user.')
+          setMessageStyle(styles.messageError)
+        } else {
+          navigate('ConfirmResetCode', {
+            email: email,
+          })
```

Comment:
Make this another if statement, (if statusCode == 200), then go to ConfirmResetCode. Then the else set's the error message to be something like `Error: Something went wrong - ${data.message}`